### ISSUE TITLE
Bug Report Title (CHANGE THIS)

### ISSUE THREAD

**Issue by ImCocoPuff** [ownagepe/Updates#27]
**Bug Description:** A simple description on what the bug is, and how it's a bug. 

**Servers Affected:** The servers affected by the bug. 

**How to reproduce the bug:** Step by step list on how you peform the exploit / bug / glitch.

**Proof of Bug:** Any link that shows / or has the bug being performed.

**Comment by ImCocoPuff** [comment]
I lost my rank after changing my name. My original name was TanIntegral702 and have been playing since last year on this server. I decided to get a rank about a week ago and when I did there was no problem. Later on I decided to change my name since it was too long to ImCocoPuff. My Supreme rank and NERD tag was removed when I changed my name. I hope to get my rank back and I hope I could enjoy playing in the server again.
